src/arb/acb.jl

```diff
@@ -69,10 +69,6 @@ function deepcopy_internal(a::AcbFieldElem, dict::IdDict)
   return b
 end
 
-function canonical_unit(x::AcbFieldElem)
```

Comment:
I am not sure if Acb (and the other three arb/acb-based fields) work well with the `is_zero` in the generic implementation, or if they rather need some `contains_zero` etc.

X-ref https://github.com/Nemocas/AbstractAlgebra.jl/pull/2069#discussion_r2073307552

Reply:
Good point.

I am not even sure if `canonical_unit` makes sense for those at all. But better to leave them then as they are.